fabric/src/main/java/me/fallenbreath/conditionalmixin/api/util/fabric/VersionCheckerImpl.java

```diff
@@ -1,20 +1,29 @@
-package me.fallenbreath.conditionalmixin.api.util;
+package me.fallenbreath.conditionalmixin.api.util.fabric;
 
 import me.fallenbreath.conditionalmixin.ConditionalMixinMod;
+import net.fabricmc.loader.api.FabricLoader;
+import net.fabricmc.loader.api.ModContainer;
 import net.fabricmc.loader.api.Version;
 
 import java.lang.reflect.Method;
-import java.util.Collection;
+import java.util.Optional;
 
-public class VersionChecker
-{
-	/**
-	 * Use fabric-loader's util to check if a given version satisfy a version predicate
-	 * @param version A Version object from fabric. You can use {@link Version#parse} to create one
-	 * @param versionPredicate A string indicates a version predicate, e.g. ">=1.2.0" or "2.0.x"
-	 */
-	public static boolean doesVersionSatisfyPredicate(Version version, String versionPredicate)
-	{
+public class VersionCheckerImpl {
```

Comment:
missing newline. keep the styling. same for other VersionCheckerImpl

Reply:
sorry, missed that one